bin/subiquity-service

```diff
@@ -5,7 +5,18 @@ export PYTHONPATH=$SNAP/lib/python3.8/site-packages
 if [ -n "$1" ]; then
    port=$1
 fi
+
+# stops some dmesg from overwriting the TUI
 /bin/dmesg -n 1
+
+# stop the systemd service completion messages from doing the same
+mkdir -p /etc/systemd/system.conf.d/
+cat > /etc/systemd/system.conf.d/subiquity.conf <<EOF
+[Manager]
+ShowStatus=no
+EOF
+systemctl daemon-reload
```

Comment:
it appears (from man systemd) you can also do this via sending SIGRTMIN+21 to pid 1, which sounds less disruptive than daemon-reload. Could you give that a try?

Reply:
Works, implemented.

Reply:
actually, I tested incorrectly.  Will follow-up with a conclusion.

Reply:
OK, per manpage, your solution looks like it should be better, but it does not appear to actually work.  The original solution does.

Reply:
Marking resolved as other kill invocations are producing the desired result, and we're deciding on the preferred version of that invocation.